### ISSUE TITLE
Remove hard coded value for holdings

### ISSUE THREAD

**Issue by borsna** [MetadataTransform/ddi-xslt#20]
Hard coded values should be avoided.

holdings:
https://github.com/MetadataTransform/ddi-xslt/blob/767774ac3c1add9fd944ed40e412ed1968c2fdda/transformations/cessda-eqb/from-ddi-2.5/ddi_2_5-cessda_eqb.xslt#L85

qstn@xml:lang:
https://github.com/MetadataTransform/ddi-xslt/blob/master/transformations/cessda-eqb/from-ddi-2.5/ddi_2_5-cessda_eqb.xslt#L170

labl@xml:lang:
https://github.com/MetadataTransform/ddi-xslt/blob/master/transformations/cessda-eqb/from-ddi-2.5/ddi_2_5-cessda_eqb.xslt#L209 
var@xml:lang:
https://github.com/MetadataTransform/ddi-xslt/blob/master/transformations/cessda-eqb/from-ddi-2.5/ddi_2_5-cessda_eqb.xslt#L136 

Values should come for source metadata or via `xsl:param`

**Comment by borsna** [comment]
@zibertg could you take a look at these cases?
The XSLT works fine but i think some hard coded values should be fixed so the XSLT works in a more general way for any DDI 2.5 XML-file.

Best regards,
Olof

**Comment by zibertg** [comment]
Agreed, for reuse of the XSLT hardcoded fields should be removed. Uploaded the updated XSLT.
